src/main/webapp/app/modules/login/login-form.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import { Button, Checkbox, Flex, Text, TextField, View, Link, Form } from '@adobe/react-spectrum';
+import Alert from '@spectrum-icons/workflow/Alert';
+import { Translate } from 'react-jhipster';
+
+export interface ILoginProps {
+  loginError: boolean;
+  handleLogin: (username: string, password: string, rememberMe: boolean) => void;
+}
+export const LoginForm = (props: ILoginProps) => {
+  const [username, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const [rememberMe, setRememberMe] = useState(false);
+  const [emptyFieldError, setEmptyFieldError] = useState(false);
+  const { handleLogin } = props;
+  const { loginError } = props;
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    if (username && password) {
+      setEmptyFieldError(false);
+      handleLogin(username, password, rememberMe);
+    } else {
+      setEmptyFieldError(true);
+    }
+  };
+
+  return (
+    <Flex alignItems="center" justifyContent="center" marginX="size-300">
+      <Form width="size-4600" isRequired isQuiet onSubmit={handleSubmit} data-testid="login-form">
+        <View marginBottom="size-600">
+          <span className="spectrum-Heading spectrum-Heading--sizeXXL">
+            <Text>
+              <Translate contentKey="login.title">Login</Translate>
+            </Text>
+          </span>
+        </View>
+        <TextField
+          width="100%"
+          marginBottom="size-300"
+          label="Email Address"
```

Comment:
How does translation work for this?

Reply:
Yah thats a good question. i have looked into the documentation and understood that we have pass the translated string.